src/data/filters.js

```diff
@@ -0,0 +1,92 @@
+export const filters = {
```

Comment:
You can fetch the filters from db/filters.json and use it here right? Any reason to re define all of it again here?

Reply:
Yeah I was testing something and I forgot to remove it will delete this